tests/test_loader.py

```diff
@@ -709,28 +746,28 @@ def test_nested_load_item(self):
 
 
 class SelectJmesTestCase(unittest.TestCase):
-        test_list_equals = {
```

Comment:
:lipstick: I would avoid these indentation fixes, being unrelated to the actual changes.

Reply:
Fixed, thanks :)